lua-linter/action.yml

```
@@ -0,0 +1,47 @@
+name: "Scalingo Lua linter GitHub Action"
+description: "Lint Lua codebases with lua-language-server"
+
+inputs:
+  working-directory:
+    description: "Directory to lint (full repository recommended)"
+    required: false
+    default: "."
+  checklevel:
+    description: "lua-language-server --checklevel (Error, Warning, Information, Hint)"
+    required: false
+    default: "Hint"
+  lua-path:
+    description: "Optional LUA_PATH to export before linting"
+    required: false
+    default: ""
+  lua-language-server-version:
+    description: "lua-language-server version to install (use 'latest' for the newest release)"
+    required: false
+    default: "latest"
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v6
+      with:
+        fetch-depth: 0
+
+    - name: Restore lua-language-server cache
+      uses: actions/cache@v4
+      with:
+        path: ~/.cache/lua-language-server
+        key: lua-language-server-${{ runner.os }}-${{ inputs.lua-language-server-version }}
+
+    - name: Install lua-language-server
+      shell: bash
+      env:
+        LUALS_VERSION: ${{ inputs.lua-language-server-version }}
+      run: bash "${GITHUB_ACTION_PATH}/scripts/install-lua-language-server.sh"
+
+    - name: Lint Lua codebase
+      shell: bash
+      working-directory: ${{ inputs.working-directory }}
+      env:
+        LUA_PATH: ${{ inputs.lua-path }}
+        CHECKLEVEL: ${{ inputs.checklevel }}
+      run: bash "${GITHUB_ACTION_PATH}/scripts/run-lua-lint.sh"
```

question: what is the purpose of prefixing these `run` property with `bash`? You already explicit that it's Bash in the `shell`property
```suggestion
      run: "${GITHUB_ACTION_PATH}/scripts/install-lua-language-server.sh"

    - name: Lint Lua codebase
      shell: bash
      working-directory: ${{ inputs.working-directory }}
      env:
        LUA_PATH: ${{ inputs.lua-path }}
        CHECKLEVEL: ${{ inputs.checklevel }}
      run: "${GITHUB_ACTION_PATH}/scripts/run-lua-lint.sh"
```